src/elements/content-explorer/GridViewItem.scss

```diff
@@ -0,0 +1,19 @@
+.bdl-GridViewItem-itemThumbnail {
+    margin-bottom: 10px;
+    padding-bottom: 65%;
```

Comment:
We should avoid relative values for spacing, as they will not be consistent across viewport sizes.